Tasks/Maven/task.json

```diff
@@ -56,18 +61,60 @@
             "helpMarkDown": "Relative path from the repository root to the Maven POM file."
         },
         {
+            "name": "goals",
+            "type": "string",
+            "label": "Goal(s)",
+            "defaultValue": "package",
+            "required": false
+        },
+        {
             "name": "options",
             "type": "string",
             "label": "Options",
             "defaultValue": "",
             "required": false
         },
         {
-            "name": "goals",
-            "type": "string",
-            "label": "Goal(s)",
-            "defaultValue": "package",
-            "required": false
+            "name": "selectSources",
+            "type": "pickList",
+            "label": "Select Sources",
+            "defaultValue" : "PomSources",
+            "groupName": "sourcesAndAuth",
+            "helpMarkDown": "Select the authentication",
+            "options": {
+                "PomSources": "Use only sources in pom.xml",
+                "OverrideSources": "Customize"
+            }
+        },
+        {
+            "name": "vstsFeedNames",
+            "type": "connectedService:Generic",
+            "label": "Include packages from this VSTS feed",
+            "required": false,
+            "defaultValue": "",
+            "groupName": "sourcesAndAuth",
+            "helpMarkDown": "Comma separated list of feeds associated with this account",
```

Comment:
This whole input needs to be updated.  There's going to be a new control that will allow for the set of feeds to be listed here with a check-box as to whether they are enabled for this build.